intelmq/bots/BOTS

```diff
@@ -500,6 +500,13 @@
             "module": "intelmq.bots.experts.gethostbyname.expert",
             "parameters": {}
         },
+        "IDEA": {
+            "description": "Converts events into the IDEA format.",
+            "module": "intelmq.bots.experts.idea.expert",
+            "parameters": {
+                "test_mode": false
```

Comment:
AFAIU the test mode just changes the category to test. What do you need that for in this bot?

We have a test type and taxonomy for this purpose.